odl/discr/grid.py

```diff
@@ -256,6 +256,16 @@ def max_pt(self):
         """
         return np.array([vec[-1] for vec in self.coord_vectors])
 
+    @property
+    def ideg(self):
+        """Indices of the degenerate axes of this grid."""
+        return self.__ideg
+
+    @property
+    def inondeg(self):
+        """Indices of the non-degenerate axes of this grid."""
+        return self.__inondeg
+
```

Comment:
Not sure we need these public. If we don't want to use the protected ones directly, I guess we can make underscored properties also.